int8 fwd_left
int8 fwd_right
int8 vert_fore
int8 vert_aft
int8 horz_fore
int8 horz_aft
